WORKSPACE

```diff
@@ -605,56 +605,50 @@ rpm(
 )
 
 rpm(
-    name = "binutils-0__2.35.2-43.el9.aarch64",
-    sha256 = "6814f37a59d417143be795b8435df1127ed0a71e86d74d00120007f694f085a2",
+    name = "binutils-0__2.35.2-48.el9.aarch64",
+    sha256 = "856c88c1e0dddb28b4994eadfbdc88b11a4367fe9c329a3f9cd07ab5d2c21b34",
     urls = [
-        "http://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os/Packages/binutils-2.35.2-43.el9.aarch64.rpm",
-        "https://storage.googleapis.com/builddeps/6814f37a59d417143be795b8435df1127ed0a71e86d74d00120007f694f085a2",
```

Comment:
This PR appears to remove all the `googleapi.com` mirrors, we need to keep those.
I've seen that happen before when running `make rpm-deps`, I'm not sure what needs to be done differently...
@dhiller any insight?
Thanks

Reply:
These get added by an automatic PR once the artifacts are mirrored to the GCS bucket. These newer rpms have to be merged first and then the periodic mirror job will upload them and create a PR.